crypto/src/main/kotlin/web5/sdk/crypto/AwsKeyManager.kt

```diff
@@ -118,24 +117,25 @@ public class AwsKeyManager @JvmOverloads constructor(
    * Retrieves the public key associated with a previously stored private key, identified by the provided alias.
    *
    * @param keyAlias The alias referencing the stored private key.
-   * @return The associated public key in JWK (JSON Web Key) format.
+   * @return The associated public key in Jwk (JSON Web Key) format.
    * @throws [AWSKMSException] for any error originating from the [AWSKMS] client
    */
-  override fun getPublicKey(keyAlias: String): JWK {
+  override fun getPublicKey(keyAlias: String): Jwk {
     val getPublicKeyRequest = GetPublicKeyRequest().withKeyId(keyAlias)
     val publicKeyResponse = kmsClient.getPublicKey(getPublicKeyRequest)
     val publicKey = convertToJavaPublicKey(publicKeyResponse.publicKey)
 
     val algorithmDetails = getAlgorithmDetails(publicKeyResponse.keySpec.enum())
     val jwkBuilder = when (publicKey) {
-      is ECPublicKey -> ECKey.Builder(JwaCurve.toJwkCurve(algorithmDetails.curve), publicKey)
+      is ECPublicKey -> {
+        val key = ECKey.Builder(JwaCurve.toNimbusCurve(algorithmDetails.curve), publicKey).build()
+        Jwk.Builder("EC", key.curve.name)
```

Comment:
Does this defs populate algorithm, keyID, keyUse?

Reply:
`alg` and `kid` is now not populated by default - relevant comment [here](https://github.com/TBD54566975/web5-spec/pull/130#discussion_r1541332623)

`kid` can be computed when needed via `Jwk#computeThumbprint()`

`keyUse` hmm... i'm pretty sure we're also not populating these by default, and it is optional per JWK spec
https://datatracker.ietf.org/doc/html/rfc7517#section-4.2

> Values defined by this specification are:
>
>   o  "sig" (signature)
>   o  "enc" (encryption)
>
>   Other values MAY be used.  The "use" value is a case-sensitive
>   string.  Use of the "use" member is OPTIONAL, unless the application
>   requires its presence.

Reply:
the only fields that are required to build a Jwk are kty and crv